public/main.js

```diff
@@ -7,12 +7,12 @@ let fetchData = {
   body: JSON.stringify(dataFromForm)
 }
 
-fetch('http://localhost:3000/tasks', fetchData)
+return fetch('http://localhost:3000/tasks', fetchData)
```

Comment:
You shouldn't hard-code the origin portion of the URL. You can just pass a path relative to the origin, _e.g._ `/tasks`.